src/main/java/Fibonacci/Fibonacci.java

```diff
@@ -0,0 +1,27 @@
+package Fibonacci;
+
+import java.util.ArrayList;
+
+public class Fibonacci {
+    public static void main(String[] args) {
+        int endNumber = 17;
+        calculatorFibonacci(endNumber);
+    }
+
+    public static void calculatorFibonacci(int endNumber) {
+        int i,number;
+        int[] numbers = new int[endNumber];
+        numbers[0]=0;
+        numbers[1]=1;
+        for (i =1; i <endNumber-1; i++) {
+            number=numbers[i]+numbers[i-1];
+           numbers[i+1]=number;
+        }
+        showArray(numbers);

```

Comment:
Bir fonskiyon her zaman bir is yapmasi idealdir. Buna Single Responsibility deniyor. Senin fonksiyonun iki is yapiyor; fibonnaci sayisini hesaplayip show ediyor. Bu methodu su signature ile yeniden duzenler misin?

public static int calculatorFibonacci(int endNumber) {